Source/Atomic/UI/SystemUI/DebugHud.h

```diff
@@ -106,6 +106,21 @@ class ATOMIC_API DebugHud : public Object
     /// Clear all application-specific stats.
     void ClearAppStats();
 
+    //Atomic Begin
+
+    bool GetSceneOpen() { return isSceneOpen_; }
+    void SetSceneOpen(bool isOpen) { isSceneOpen_ = isOpen; }
+
+    String GetStatsString() { return statsString_; }
+    String GetModeString() { return modeString_; }
+    String GetProfilerString() { return profilerString_; }

```

Comment:
I was having trouble setting them to const since the String values constantly keep changing.
Sending them as String& only return 0 and the reason I ended up using Strings and not just use the variables that already existed is because the widgets can only use String values when setting their text.